proto/osmosis/incentives/gov.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+package osmosis.incentives;
+
+import "gogoproto/gogo.proto";
+import "osmosis/incentives/group.proto";
+
+option go_package = "github.com/osmosis-labs/osmosis/v19/x/incentives/types";
+
+// CreateGaugeGroupsProposal is a type for creating one or more gauge groups via
+// governance. This is useful for creating gauge groups without having to pay
+// creation fees.
+message CreateGaugeGroupsProposal {
+  option (gogoproto.goproto_getters) = false;
+  option (gogoproto.goproto_stringer) = false;
+
+  string title = 1;
+  string description = 2;
+
+  repeated osmosis.incentives.CreateGroup create_groups = 3
+      [ (gogoproto.nullable) = false ];
```

Comment:
Why don't we insert pool_ids here directly? What is the reason for having another data structure?

Reply:
1. You can't represent a 2D array without a separate message in protobuf
2. Allows for extending once we need to provide more than just pool-ids

Reply:
Right, this is a multi-group one